server/routes/bulkThreads.ts

```diff
@@ -51,7 +51,7 @@ const bulkThreads = async (
         threads.url, threads.pinned, topics.id AS topic_id, topics.name AS topic_name,
         topics.description AS topic_description, topics.chain_id AS topic_chain,
         topics.telegram AS topic_telegram,
-        topics.community_id AS topic_community, collaborators, chain_entities
+        topics.community_id AS topic_community, collaborators, chain_entities, linked_threads
```

Comment:
Nope.

/bulkOffchain didn't get replaced by a smarter query. We just weren't using it anymore. (Because we switched all the burden to this bulkThreads fetch.) 

I agree though, manual SQL like this is a nightmare to maintain, and this exact route is a big part of the reason I've been pushing for a new comment root_id system for like... over a year now lol.